docs/README.md

```diff
@@ -36,19 +36,52 @@ As per the [Team Data Science Process (TDSP)](https://docs.microsoft.com/en-us/a
 
 An important design goal is modularity: the components provide utility on their own, and can be re-used in different scenarios to solve other problems at a lower incremental cost.
 
-# Scenario (business understanding)
+# Scenario
 
-TBD
+The solution deals with an imaginary IoT-enabled manufacturing environment comprised of generalized rotational equipment, which may include pumps, turbines, gearboxes, compressors, and engines. 
 
-# Data acquisition and understanding
+The machines are equipped with sensors from which telemetry is regularly transmitted to the cloud in near real time. Maintenance logs are also available and, among other things, contain records of failure events indicating exact points in time when a machine had a critical failure of a particular type.
+
+The main business objective of Predictive Maintenance is preempting these events in the future by raising alarms in advance, so that appropriate preventive activities can be carried out.
+
+# Data acquisition
+
+To enable modeling, data has to be ingested into the target analytic environment.
+
+The solution provides two data acquisition options:
+
+1. Generation of static seed data
+2. Ingestion of IoT telemetry and enterprise data (maintenance logs)
+
+As shown in the diagram below, both options produce a data set of the same format. Option #1 is available as a "shortcut" for quickly (in a matter of minutes) generating an arbitrarily large input data set, whereas option #2 emulates a realistic production situation where incoming telemetry is staged over an extended period of time in storage blobs, and maintenance logs are accumulated in some other data store (exemplified as a SQL database in the diagram).
 
 ![](img/data_collection.png)
 
+The data generator included with the solution is capable of both generating static seed data (option #1) and emitting real-time telemetry (option #2).
+
```

Comment:
To better understand IoT data collection in a production space, review [the documentation for the IoT Remote Monitoring solution](https://docs.microsoft.com/en-us/azure/iot-accelerators/iot-accelerators-remote-monitoring-explore).

Reply:
How is this helpful?

Reply:
We don't need to reinvent the tactics to collect real-world telemetry with IoT.  The customer may have a few questions about the data generator that we can answer, but when they are ready to use real-world data we should refer the customer to the documentation that has already been successful.

Reply:
Let's add this in the productionalization document. Don't want this intro to be too heavy.

Reply:
I can live with that.